src/templates/one/content.js

```diff
@@ -0,0 +1,37 @@
+const I18n = require('frontend-i18n');
+
+const getTranslationsForSite = siteId => {
```

Comment:
Should it be in a "utils" folder, right? (I mean, not specific for this template)

Reply:
yeah, totally! this is just for _POCing_

Reply:
Great 😁